HttpStress is not running on Windows in CI

```
##[error]Failed to request agent. Exception Image Build.Windows.Amd64.VS2022.Pre doesn't exist in pool NetCore1ESPool-Public
```

https://dev.azure.com/dnceng/public/_build/results?buildId=1877931&view=logs&j=2d2b3007-3c5c-5840-9bb0-2b1ea49925f3

@dotnet/dnceng any idea what happenned to this image and how to address this problem?

--- a/eng/pipelines/libraries/stress/http.yml
+++ b/eng/pipelines/libraries/stress/http.yml
@@ -97,7 +97,7 @@ jobs:
     DUMPS_SHARE_MOUNT_ROOT: "C:/dumps-share"
   pool:
     name: NetCore1ESPool-Public
-    demands: ImageOverride -equals Build.Windows.Amd64.VS2022.Pre
+    demands: ImageOverride -equals Build.Windows.Amd64.VS2022.Pre.Open
 
   steps:
   - checkout: self

--- a/eng/pipelines/libraries/stress/ssl.yml
+++ b/eng/pipelines/libraries/stress/ssl.yml
@@ -55,7 +55,7 @@ jobs:
   timeoutInMinutes: 120
   pool:
     name: NetCore1ESPool-Public
-    demands: ImageOverride -equals Build.Windows.Amd64.VS2022.Pre
+    demands: ImageOverride -equals Build.Windows.Amd64.VS2022.Pre.Open
 
   steps:
   - checkout: self

